contributed_definitions/nyaml/NXmpes_xps.yaml

```diff
@@ -0,0 +1,192 @@
+              ]
+      depends_on(NX_CHAR):
+        exists: recommended
+        doc: "Reference to the transformation describing the orientation of the analyzer relative to the beam."
+      (NXtransformations):
+        exists: recommended
+        doc: "Set of transformations, describing the relative orientation of the analyzer with respect to the beam coordinate system."
+    (NXprocess):
+        energy_calibration(NXcalibration):
+          exists: optional
+          calibration_file(NXnote):
+            exists: optional
+            doc: "Name of the calibration file containing the calibration data."     
+        intensity_calibration(NXcalibration):
+          exists: optional
+          applied(NX_BOOLEAN):
+            doc: "Has an intensity calibration been applied? 
+              That is, has the transmission function of the analyzer been taken into account?"
+          transmission_function(NX_FLOAT):
+            exists: recommended
+            doc: "Transmission function of the electron collection column.
+              This should be a link to /entry/instrument/collectioncolumn/transmission_function." 
+            units: NX_ANY  
+            file(NXnote):
+              exists: recommended
+              doc: "Name of the file containing the transmission function data."
+        energy_reference(NXcalibration):
+          exists: optional
+          applied(NX_BOOLEAN):
+            doc: "Have the energy axes been adjusted (e.g., in cases where samples are not conductive)?"
+          emission_line(NX_CHAR):
+            exists: recommended
+            doc: "Emission line which was used for the calibration."
+          offset(NX_FLOAT):
+            exists: recommended
+            doc: "Offset between measured binding energy and calibrated binding energy of the emission line."
+          binding_energy(NX_FLOAT):
+            exists: recommended
+            doc: "The binding energy that the specified emission line appeared at,
+              after adjusting the binding energy scale, in units of eV"            
+          calibrated_axis(NX_FLOAT):
+            exists: recommended
+            doc: "This is the calibrated energy axis to be used for data plotting.
+              This should be a link to /entry/data/energy."
+    (NXsample):     
+      form(NX_CHAR):
+        exists: optional
+        enumeration:
+          # There should be more options here.
+          [
+            "foil",
+            "single crystal",
+            "pellet",
+            "powder",
+          ]
+      gas_pressure(NX_FLOAT):
+        unit: NX_PRESSURE
+        doc: |
+          In the case of a fixed pressure measurement this is the scalar pressure.
+          In the case of an experiment in which pressure changes, or anyway it is recorded,
+          this is an array of length m of pressures.
+      gas_phase(NX_CHAR):
+        exists: optional
+        doc: "Description of the constituent gases of the gas phase."    
```

Comment:
how about using NXsample/gas ?

Reply:
Yes, that makes sense. However, for me sample and sample environment are two distinct entities, so maybe this should be split at some point.